aphrodite/aphrodite.d.ts

```diff
@@ -69,6 +70,30 @@ declare module "aphrodite" {
     }
 
     export var StyleSheetTestUtils: StyleSheetTestUtilsStatic;
+
+    export interface SelectorHandler {
+        (selector: string, baseSelector: string, callback: (selector: string) => string): string;
+    }
+
+    export interface Extension {
+        selectorHandler?: SelectorHandler,
```

Comment:
Yep! This would probably make more sense if there were other kinds of extensions, where you'll have either a `selectorHandler` property or something else. But this is correct semantically (if you pass in an empty object it will handle it by doing nothing).